app/src/main/java/com/mymeetings/android/model/managers/CalendarEventAlertManager.kt

```diff
@@ -21,22 +22,50 @@ class CalendarEventAlertManager(private val clockUtils: ClockUtils) {
 
     private fun getAlertTypeForCalendarEvent(calendarEvent: CalendarEvent) : CalendarEventWithViewAlert {
 
-        val viewAlert = when {
+        val viewAlert = getViewAlertType(calendarEvent)
+        val relativeTime = getRelativeTime(calendarEvent.startTime)
+        val startToEndTime = getEventStartToEndTime(calendarEvent.startTime, calendarEvent.endTime)
+        return CalendarEventWithViewAlert(
+            calendarEvent,
+            viewAlert,
+            relativeTime,
+            startToEndTime
+        )
+    }
+
+    private fun getViewAlertType(calendarEvent: CalendarEvent) : ViewAlertType {
+        return when {
             calendarEvent.startTime < clockUtils.currentTimeMillis() -> {
                 ViewAlertType.RUNNING
             }
-            calendarEvent.startTime > clockUtils.currentTimeMillis() + priorityTime
-                    && calendarEvent.priority.value >= CalendarEventPriority.MEDIUM.value -> {
+            calendarEvent.startTime < clockUtils.currentTimeMillis() + priorityTime -> {
```

Comment:
We don't need two types of time configurations (priority and normal) for the first release. Just 10 minutes will do.  
Also we need to make this user configurable in the future release.